app/main.py

```diff
@@ -1,25 +1,68 @@
 from abc import ABC
+from pyclbr import Class
```

Comment:
The import `from pyclbr import Class` is incorrect for the intended usage. `Class` from `pyclbr` is not meant to be used as a type for instantiation. Consider using `type` or a specific class type that can be instantiated.